pubspec.yaml

```diff
@@ -7,21 +7,20 @@ documentation: https://nyxx.l7ssha.xyz
 issue_tracker: https://github.com/nyxx-discord/nyxx/issues
 
 environment:
-  sdk: '>=3.0.0 <4.0.0'
+  sdk: ">=3.0.0 <4.0.0"
```

Comment:
single quotes my beloved :(

Reply:
VSCode autoformatting hates me for some reason :/